src/OmniSharp.Abstractions/Services/IWorkspaceOptionsProvider.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.CodeAnalysis.Options;
+
+namespace OmniSharp.Services
+{
+    public interface IWorkspaceOptionsProvider
+    {
+        OptionSet Process(OptionSet workOptionSet);
```

Comment:
I did a bit of a double-take when I read `workOptionSet`. I wasn't sure if this meant "working option set" or "workspace option set". Maybe just `optionSet` to avoid confusion?